app/views/dashboard/service_calendars/_view_full_calendar.html.haml

```diff
@@ -89,15 +89,15 @@
                 %h4.panel-title
                   = t(:dashboard)[:service_calendars][:consolidated_request][:services][:header]
               .panel-body
-                - if @protocol.has_per_patient_per_visit? @service_request, @portal
+                - if @protocol.has_line_items_of_type? @service_request, @portal
                   - @protocol.arms.each do |arm|
                     .arm_calendar_container.calendar-container{ data: { arm_id: arm.id } }
                       - next if arm.visit_groups.empty?
                       = render partial: '/dashboard/service_calendars/master_calendar/pppv/pppv_calendar', locals: { service_request: @service_request, arm: arm, page: @pages[arm.id], pages: @pages, review: false, merged: true, study_tracker: false }
-                - if @protocol.has_one_time_fees? @service_request, true
+                - if @protocol.has_line_items_of_type? @service_request, portal = true, one_time_fee = true
```

Comment:
We are assigning them this way, so that you can tell from here what the true value is doing, instead of having to dig up the method and check.

The portal value was already being passed into this method pre-refactor. The method is in the protocol model, which most certainly does not have access to @portal instance variable, so I'm not sure what you're saying.